tests/integration/utils/named-list.test.ts

```diff
@@ -0,0 +1,120 @@
+import { expect, test, describe } from "vitest";
+import { render, screen } from "@testing-library/svelte";
+import NamedList from "$lib/components/composites/utils/NamedList.svelte";
+// @ts-expect-error "Snippets have implicetly type any"
```

Comment:
This is unused
```suggestion
```

Reply:
Good catch thank, a relic of the old days when they were not yet correctly typed.

Reply:
This works!